include/libtup.h

```diff
@@ -110,13 +110,13 @@ typedef struct
 
 /**
  * \ingroup message
- * Get/Set sensor value structure
+ * Get/Set input value structure
  */
 typedef struct
 {
-    uint8_t sensor_id;        /**< sensor id */
-    uint16_t sensor_value;    /**< sensor value */
-} TupSensorValueArgs;
+    uint8_t input_id;        /**< input id */
+    uint16_t input_value;    /**< input value */
```

Comment:
you shall declare it as a int32_t to correspond to libart

Reply:
Done